packages/dds/merge-tree/src/client.ts

```diff
@@ -252,15 +253,20 @@ export class Client extends TypedEventEmitter<IClientEvents> {
 	 * Obliterates the range. This is similar to removing the range, but also
 	 * includes any concurrently inserted content.
 	 *
-	 * @param start - The inclusive start of the range to obliterate
-	 * @param end - The exclusive end of the range to obliterate
+	 * @param start - The start of the range to obliterate. Inclusive if the side is Before
+	 * @param end - The end of the range to obliterate. Inclusive if the side is After
 	 */
+	// eslint-disable-next-line import/no-deprecated
 	public obliterateRangeLocal(
-		start: number,
-		end: number,
+		start: SequencePlace,
```

Comment:
we shouldn't use sequence place,  we should use `number | InteriorSequencePlace`, as we don't have and don't plan to add testing for `start | end`